accountsdb-plugin-manager/src/accountsdb_plugin_service.rs

```diff
@@ -170,13 +170,21 @@ impl AccountsDbPluginService {
         let libpath = result["libpath"]
             .as_str()
             .ok_or(AccountsdbPluginServiceError::LibPathNotSet)?;
+        let mut libpath = PathBuf::from(libpath);
+        if libpath.is_relative() {
+            let config_dir = accountsdb_plugin_config_file
+                .parent()
+                .expect("failed to resolve dir of accounts plugin config");
```

Comment:
Nits pick: "Failed to resolve". Outputting the config file path probably helps debugging.

Reply:
done